internal/worker/uniter/runner/context/env_test.go

```diff
@@ -254,119 +243,6 @@ func (s *EnvSuite) TestEnvUbuntu(c *gc.C) {
-		centosVars := []string{
-			"LANG=C.UTF-8",
-			"PATH=path-to-tools:foo:bar",
-		}
-
-		if testBase.Channel.Track == "7" {
-			centosVars = append(centosVars, "TERM=screen-256color")
-		} else {
-			centosVars = append(centosVars, "TERM=tmux-256color")
-		}
-
-		environmenter := context.NewRemoteEnvironmenter(
-			func() []string { return []string{} },
-			func(k string) string {
-				switch k {
-				case "PATH":
-					return "foo:bar"
-				}
-				return ""
-			},
-			func(k string) (string, bool) {
-				switch k {
-				case "PATH":
-					return "foo:bar", true
-				}
-				return "", false
-			},
-		)
-
-		hookContext, contextVars := s.getHookContext(c, false, state, unit)
-		paths, pathsVars := s.getPaths()
-		actualVars, err := hookContext.HookVars(stdcontext.Background(), paths, false, environmenter)
-		c.Assert(err, jc.ErrorIsNil)
-		s.assertVars(c, actualVars, contextVars, pathsVars, centosVars)
-
-		relationVars := s.setRelation(hookContext)
-		secretVars := s.setSecret(hookContext)
-		actualVars, err = hookContext.HookVars(stdcontext.Background(), paths, false, environmenter)
-		c.Assert(err, jc.ErrorIsNil)
-		s.assertVars(c, actualVars, contextVars, pathsVars, centosVars, relationVars, secretVars)
-	}
-}
-
-func (s *EnvSuite) TestEnvGenericLinux(c *gc.C) {
-	ctrl := gomock.NewController(c)
-	defer ctrl.Finish()
-
-	state := api.NewMockUniterClient(ctrl)
-	state.EXPECT().StorageAttachment(names.NewStorageTag("data/0"), names.NewUnitTag("this-unit/123")).Return(params.StorageAttachment{
-		Kind:     params.StorageKindBlock,
-		Location: "/dev/sdb",
-	}, nil).AnyTimes()
-	unit := api.NewMockUnit(ctrl)
-	unit.EXPECT().Tag().Return(names.NewUnitTag("this-unit/123")).AnyTimes()
-
-	s.PatchValue(&jujuos.HostOS, func() ostype.OSType { return ostype.GenericLinux })
-	s.PatchValue(&jujuversion.Current, version.MustParse("1.2.3"))
-
-	genericLinuxVars := []string{
-		"LANG=C.UTF-8",
-		"PATH=path-to-tools:foo:bar",
-		"TERM=screen",
-	}
-
-	environmenter := context.NewRemoteEnvironmenter(
-		func() []string { return []string{} },
-		func(k string) string {
-			switch k {
-			case "PATH":
-				return "foo:bar"
-			}
-			return ""
-		},
-		func(k string) (string, bool) {
-			switch k {
-			case "PATH":
-				return "foo:bar", true
-			}
-			return "", false
-		},
-	)
-
-	hookContext, contextVars := s.getHookContext(c, false, state, unit)
-	paths, pathsVars := s.getPaths()
-	actualVars, err := hookContext.HookVars(stdcontext.Background(), paths, false, environmenter)
-	c.Assert(err, jc.ErrorIsNil)
-	s.assertVars(c, actualVars, contextVars, pathsVars, genericLinuxVars)
-
-	relationVars := s.setRelation(hookContext)
-	secretVars := s.setSecret(hookContext)
-	actualVars, err = hookContext.HookVars(stdcontext.Background(), paths, false, environmenter)
-	c.Assert(err, jc.ErrorIsNil)
-	s.assertVars(c, actualVars, contextVars, pathsVars, genericLinuxVars, relationVars, secretVars)
-}
-
 func (s *EnvSuite) TestHostEnv(c *gc.C) {
```

Comment:
How is this any different to TestHostEnvUbuntu now?

Reply:
No difference, good spot